src/encoded/static/components/biosample.js

```diff
@@ -71,7 +71,7 @@ class BiosampleComponent extends React.Component {
                     </div>
                     <ItemAccessories item={context} audit={{ auditIndicators: this.props.auditIndicators, auditId: 'biosample-audit' }} />
                 </header>
-                {this.props.auditDetail(context.audit, 'biosample-audit', { session: this.context.session, except: context['@id'] })}
+                {this.props.auditDetail(context.audit, 'biosample-audit', { session: this.context.session, sessionProperties: this.context.session_properties, except: context['@id'] })}
```

Comment:
If you like, you can delete the `except` property whereever it gets passed to `auditDetail` because `auditDetail` doesn’t use it anymore. It’s harmless to pass in; just unneeded, so it’s up to you.

Reply:
Would you have an issue if I keep it? There are 30 files modified and a lot of changes... Removing it when it is actually needed may cause a hard-to-detect problem?

Reply:
No issues. It’s harmless. It could be in a later audit ticket.